components/brave_rewards/resources/internals/BUILD.gn

```diff
@@ -1,9 +1,12 @@
 import("//brave/components/common/typescript.gni")
 
 transpile_web_ui("internals") {
-  entry_points = [
-    ["brave_rewards_internals", rebase_path("brave_rewards_internals.tsx")]
-  ]
+  entry_points = [ [
+        "brave_rewards_internals",
+        rebase_path("brave_rewards_internals.tsx"),
+      ] ]
+
+  deps = [ "//brave/vendor/bat-native-ledger/include/bat/ledger/public/interfaces:ledger_types_js" ]
```

Comment:
I think this won't pass the formatting check, but maybe it's more lenient than I recall.

Reply:
I ran `npm run lint` and committed the formatting changes - it didn't touch this line.

Reply:
That said, it does seem a little bit too long...